utils/types.d.ts

```diff
@@ -0,0 +1,23 @@
+export interface IEvent {
+  id?: string
+  title?: string
+  description?: string
+  date?: string
+  type?: string
+}
+
+export interface IProvider {
+  id?: string
+  commercial_name?: string
+  address?: string
+  sales_phone?: string
+  b2b_phone?: string
+  sales_email?: string
+  b2b_email?: string
+  legal_name?: string
+  details?: string
+  latitude?: string
+  longitude?: string
+  is_active?: boolean
+  logo_url?: string
+}
```

Comment:
I think at lease ID and title/sales_email should be not null

Reply:
The issue is, Typescript displays an error because, I use the `IProvider` to define the form state, so, whem I create a new provider the id is null, because I'm creating it yet

Reply:
Okok, it's good to me. Thanks for making it clear